bin/cli.ml

```diff
@@ -53,6 +53,13 @@ let opam =
   let docv = "FILE" in
   Arg.(value & opt (some path_arg) None & info [ "opam" ] ~doc ~docv)
 
+let token =
+  let doc =
+    "The github token to use. If absent looks for a token stored locally."
+  in
+  let docv = "FILE" in
+  Arg.(value & opt (some path_arg) None & info [ "token" ] ~doc ~docv)
```

Comment:
Cmdliner also provides a way to pass it directly through an env variable which I think would also make sense for CI jobs so that the token don't get printed in the CI logs for instance. Could you please add that?

Reply:
good idea, done!